ingestion/src/metadata/ingestion/source/database/bigquery/metadata.py

```diff
@@ -604,18 +608,27 @@ def get_database_names(self) -> Iterable[str]:
     def get_view_definition(
```

Comment:
Let's change the method name to be relevant like `get_schema_definition` and add a doc string

Reply:
Changed the method name `get_view_definition ` ->  `get_schema_definition`